visual_identity.md

```diff
@@ -0,0 +1,228 @@
+  <div class="col-12 col-md-6"><img src="{{ site.baseurl }}/img/arrow-logo_chevrons_white-txt_black-bg.png" width="100%"></div>
+</div>
+</div>
+
+and the "Apache Arrow" logotype on its own:
+
+<div class="container justify-content-evenly center py-3">
+<div class="row">
+  <div class="col-12 col-md-6"><img src="{{ site.baseurl }}/img/arrow-logo_text_black-txt_white-bg.png" width="100%"></div>
+  <div class="col-12 col-md-6"><img src="{{ site.baseurl }}/img/arrow-logo_text_white-txt_black-bg.png" width="100%"></div>
+</div>
+</div>
+
+## Using the Apache Arrow logos {#using-the-logo}
+
+### General guidelines
+
+Usage guidelines for official logos are quite similar across many brands, and Apache Arrow is no exception. The core principle is to respect the visual integrity of the logo. Guidelines that will help you do so include:
+
+ - Please don't visually distort the logos
+ - Please reproduce at a clearly visible resolution 
+ - Please don't modify the text or the design of the chevrons
+ - Please don't add additional text to the logotype or imagery to the chevrons
+ 
+In general, the principles laid out in the [Apache Foundation Style Guide][3] for usage of the Apache logo are also appropriate for the Apache Arrow logo. If you're in a situation where you want to refer formally to the Apache Arrow project you should adhere to these guidelines pretty closely and avoid modifying the official logos.
+
+### Regarding unofficial images
+
+That said, there are occasions where you may want to use the logo unofficially. The purpose of the visual identity guidelines as described so far is to provide guidance on what to do when you *do* need to refer to Apache Arrow in an official or formal way. It's not meant to prevent people from having fun creating unofficial images when appropriate. 
+
+To provide useful guidance on when unofficial images are acceptable and unacceptable, here are two examples:
+
+<div class="container justify-content-evenly center py-3">
+<div class="row">
+  <div class="col-12 col-md-6"><img src="{{ site.baseurl }}/img/offbrand_hex_1.png" width="100%"></div>
+  <div class="col-12 col-md-6"><img src="{{ site.baseurl }}/img/offbrand_hex_2.png" width="100%"></div>
+</div>
+</div>
+
+The image on the left highlights the letter R and one of the chevrons in a fashion that is reminiscent of the R prompt, using colors that align with those used by the R-Ladies organization. Using this image informally as part of a presentation to an R-Ladies meetup would be consistent with the Apache Arrow guidelines, because it is thematically appropriate to the context and the image is not being presented as if it were the official Apache Arrow logo. The purple elements have been introduced to the image in a manner that leaves the visual integrity of the Apache Arrow logo intact, and in a fashion that reflects the informal context-specific use within an R-Ladies meetup.
```

Comment:
I think it would be good to link the first occurrence of "R-Ladies" here to https://rladies.org for the benefit of readers who might not know what R-Ladies is.